src/main/resources/assets/gofish/lang/zh_cn.json

```diff
@@ -1,115 +1,127 @@
-    "advancements.fishing.diamond_crate.title":"渔者的究极宝藏",
-    "advancements.fishing.diamond_crate.description":"钓上钻石盒子——能钓上来的最好的宝藏。",
-    "advancements.fishing.baked_seaweed.title":"草（一种食物）",
-    "advancements.fishing.baked_seaweed.description":"把没用的海苔变成香烤海苔。",
-    "advancements.fishing.astral_crate.title":"天外来盒",
-    "advancements.fishing.astral_crate.description":"在满月下钓上一个星体盒。",
-    "advancements.fishing.aquatic_astral_stew.title":"舌尖上的银河",
-    "advancements.fishing.aquatic_astral_stew.description":"用满月钓到的鱼做出星间水产煲吧。"
+  "itemGroup.gofish.group":"一起钓鱼吧！",
+
+  "gofish.crate_tooltip":"潜行右击以开箱",
+  "enchantment.gofish.deepfry":"深度烧烤",
+  
+  "item.gofish.spikerfish":"尖刺鲀",
+  "item.gofish.smokey_salmon":"熏烟鲑",
+  "item.gofish.blackstone_trout":"黑石鳟",
+  "item.gofish.bonefish":"骨鱼",
+  "item.gofish.ender_eel":"末影鳗",
+  "item.gofish.endfish":"末地鱼",
+  "item.gofish.gilded_blackstone_carp":"镶金黑石鲤",
+  "item.gofish.icicle_fish":"冰锥鱼",
+  "item.gofish.lilyfish":"莲叶鱼",
+  "item.gofish.matrix_fish":"能量矩阵鱼",
+  "item.gofish.seaweed":"海苔",
+  "item.gofish.baked_seaweed":"香烤海苔",
+  "item.gofish.seaweed_eel":"海苔鳗",
+  "item.gofish.slimefish":"黏液鱼",
+  "item.gofish.snowball_fish":"雪球鱼",
+  "item.gofish.terrafish":"地鱼",
+  "item.gofish.carrot_carp":"胡萝卜鲤",
+  "item.gofish.baked_carrot_carp":"焙煎胡萝卜鲤",
+  "item.gofish.oakfish":"橡木鱼",
+  "item.gofish.charfish":"炭鱼",
+  "item.gofish.soul_salmon":"灵魂鲑",
+  "item.gofish.magma_cod":"熔岩鳕",
+  "item.gofish.basalt_bass":"玄武岩鲈",
+  "item.gofish.obsidian_halibut":"黑曜石大比目鱼",
+  "item.gofish.rainy_bass":"雨鲈",
+  "item.gofish.cloudy_crab":"云蟹",
+  "item.gofish.thundering_bass":"雷鲈",
+  "item.gofish.smoked_cloudy_crab":"熏制云蟹",
+  "item.gofish.blizzard_bass":"霜雪鲈",
+
+  "item.gofish.steamed_bass":"清蒸雨鲈",
+  "item.gofish.steamed_bass.tooltip_1":"虽然蒸着蒸着差点就蒸发了，",
+  "item.gofish.steamed_bass.tooltip_2":"但味道还是很不错的嘛！",
+
+  "item.gofish.lunarfish":"月鱼",
+  "item.gofish.starry_salmon":"星鲑",
+  "item.gofish.galaxy_starfish":"天海星",
+  "item.gofish.nebula_swordfish":"星云剑旗鱼",
+
+  "item.gofish.aquatic_astral_stew":"星间水产煲",
+  "item.gofish.aquatic_astral_stew.tooltip_1":"是使用星间的各色水产",
+  "item.gofish.aquatic_astral_stew.tooltip_2":"制作而成的鱼汤。",
+  "item.gofish.aquatic_astral_stew.tooltip_3":"味蕾的宇宙大爆炸。",
+
+  "item.gofish.simple_lure":"简单鱼饵",
+  "item.gofish.golden_fish":"金鱼",
+  "item.gofish.soul_lure":"灵魂鱼饵",
+
+  "gofish.lure.tooltip_1":"放在身上的时候",
+  "gofish.lure.tooltip_2":"提供§2+%d§r饵钓。",
+  "gofish.lots.tooltip_2":"提供§2+%d§r幸运%s。",
+
+  "item.gofish.endfish_and_chorus":"炸末地鱼佐紫颂果",
+  "item.gofish.endfish_and_chorus.tooltip_1":"甘与咸的搭配。",
+  "item.gofish.endfish_and_chorus.tooltip_2":"话说这样的配合有点炸鱼薯条的感觉呢…？",
+  "item.gofish.baked_endfish":"盐焗末地鱼",
+  "item.gofish.baked_endfish.tooltip_1":"焗烤末地鱼的肉质",
+  "item.gofish.baked_endfish.tooltip_2":"可谓入口即化，很是诱人。",
+  "item.gofish.grilled_blackstone_trout":"炭烧黑石鳟",
+  "item.gofish.grilled_blackstone_trout.tooltip_1":"明火慢烤，咸香宜人。",
+  "item.gofish.grilled_blackstone_trout.tooltip_2": "",
```

Comment:
In-game testing showed its absence is irrelevant - The missing line will appear as a missing line, namely a line that's blank. It surely does make things a bit ugly...